spec/components/work_history_review_component_spec.rb

```diff
@@ -97,39 +99,143 @@
 
   context 'when there are breaks in the work history' do
     let(:application_form) do
-      Timecop.freeze(Time.zone.local(2019, 10, 1, 12, 0, 0)) do
-        create(:application_form) do |form|
-          data[:start_date] = Time.zone.local(2019, 8, 1)
-          data[:end_date] = Time.zone.local(2019, 9, 1)
-          form.application_work_experiences.create(data)
+      create(:application_form) do |form|
+        data[:start_date] = Time.zone.local(2019, 8, 1)
+        data[:end_date] = Time.zone.local(2019, 9, 1)
+        form.application_work_experiences.create(data)
 
-          form.work_history_breaks = 'WE WERE ON A BREAK!'
-        end
+        form.work_history_breaks = 'WE WERE ON A BREAK!'
       end
     end
 
-    context 'when work history breaks are editable' do
+    around do |example|
+      Timecop.freeze(Time.zone.local(2019, 10, 1)) do
+        example.run
+      end
+    end
+
+    context 'when capture-all work history breaks are editable' do
```

Comment:
Does this mean the "old" data? I'm not sure from the name

Reply:
Yep, trying to refer to this:

![image](https://user-images.githubusercontent.com/42817036/74849969-48511e00-5331-11ea-8e1a-122033c3dcb1.png)

Apparently, "**_consolidated_** work history breaks" is the way to go according to Paul H and Emma. 😅 Do you think that clearer?

Reply:
Sounds good!